### PR TITLE
add videojs-vast-vpaid v2.0.1 with npm auto-update

### LINKED ISSUE #8017: VideoJS Vast Vpaid Plugin
**Library name: videojs-vast-vpaid** 
**Git repository url: https://github.com/MailOnline/videojs-vast-vpaid**
**npm package url(optional):** https://www.npmjs.com/package/videojs-vast-vpaid
**License(s): FREE**
**Official homepage:http://mailonline.github.io/videojs-vast-vpaid**
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #8017
@x09326 please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/MailOnline/videojs-vast-vpaid
 * Official website (optional, not the repository): http://mailonline.github.io/videojs-vast-vpaid
 * NPM package url (optional): https://www.npmjs.com/package/videojs-vast-vpaid
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 92
   - Count of watchers: 70
   - Count of forks: 75
 * NPM download stats (optional):
   - Downloads in the last day: 12
   - Downloads in the last week: 719
   - Downloads in the last month: 1,078

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #8017, cc @carpasse

### CODE REVIEW

**Comment by x09326** [comment]
@PeterDaveHello 
I think this PR is okay, please help me review it, thanks.

**Comment by PeterDaveHello** [comment]
Don't we need `videojs_4.vast.vpaid.min.js`?

**Comment by x09326** [comment]
I think we don't need them.
https://github.com/MailOnline/videojs-vast-vpaid/issues/213

**Comment by PeterDaveHello** [comment]
I didn't see the reason to drop it, it's for different versions of videojs, should add that.

**Comment by PeterDaveHello** [comment]
May also need `ie8fix.js`, you should read the document:
https://github.com/MailOnline/videojs-vast-vpaid#integration-with-videojs-4-and-5

**Comment by kennynaoh** [comment]
@PeterDaveHello 
Sorry, I forgot to read the `README.md`.
So `es5-shim.js` this file should also be added, right?
Thanks!

**Comment by PeterDaveHello** [comment]
Don't we have `es5-shim` here https://cdnjs.com/libraries/es5-shim?

**Comment by PeterDaveHello** [comment]
Don't assign to me again if we're not going to review the content of this PR, you are still the assignee who should fix the PR first.

**Comment by kennynaoh** [comment]
Got it.
@x09326 
I have fixed the `npmFileMap` to get the correct file.
Please help me review it again.
Thanks!

**Comment by x09326** [comment]
@PeterDaveHello 
I think this PR is okay, please help me review it, thanks.

File: ajax/libs/videojs-vast-vpaid/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "videojs-vast-vpaid",
+  "filename": "videojs_5.vast.vpaid.min.js",
+  "version": "2.0.1",
+  "authors": [
+    "Carlos Serrano <carlos.pastor@mailonline.co.uk>"
+  ],
```
Comment (PeterDaveHello) [inline ajax/libs/videojs-vast-vpaid/package.json]: Single author should use string not array.

**Comment by kennynaoh** [comment]
@x09326 
I have fixed the author format.
Please help me review it again.
Thanks!

**Pull request merged**